foundation/foundation-archetype/src/main/resources/META-INF/archetype-post-generate.groovy

```diff
@@ -14,14 +14,8 @@ import javax.lang.model.SourceVersion
 final Logger logger = LoggerFactory.getLogger("com.boozallen.aissemble.foundation.archetype-post-generate")
 
 def dir = new File(new File(request.outputDirectory), request.artifactId)
-
-// code below fixes root pom. See links below
-// https://issues.apache.org/jira/browse/ARCHETYPE-587
-// https://issues.apache.org/jira/browse/ARCHETYPE-584
-file = new File(dir, "pom.xml")
```

Comment:
I: Tested and it seems these workarounds are no longer needed with the version of maven we're using